modules/installation-guide/partials/assembly_mounting-secrets-or-a-configmap-as-file-or-env-variable-in-container.adoc

```diff
@@ -0,0 +1,28 @@
+:parent-context-of-mounting-a-secret-or-a-configmap-as-a-file-or-an-environment-variable-into-a-container: {context}
+
+[id="mounting-a-secret-or-a-configmap-as-a-file-or-an-environment-variable-into-a-container_{context}"]
+= Mounting a Secret or a ConfigMap as a file or an environment variable into a {prod-short} container
+
+:context: mounting-a-secret-or-a-configmap-as-a-file-or-an-environment-variable-into-a-eclipse-che-container
+
+Secrets are {platforms-name} objects that store sensitive data such as:
+
+* usernames
+* passwords
+* authentication tokens
+* and configurations
```

Comment:
which kind of configuration you expect to be stored in secret. Probably it's config map that should be used for configuration.